pkg/universe.dagger.io/python/test/test.cue

```diff
@@ -10,50 +10,57 @@ import (
 dagger.#Plan & {
 	actions: test: {
 
-		// Run with a custom path to python
-		customPath: {
+		"python.cue": {
+			// Run with a custom path to python
+			customPath: {
 
-			// python:3.10-alpine has both python3 and python in the PATH
-			_image: docker.#Pull & {
-				source: "python:3.10-alpine"
-			}
-			run: python.#Run & {
-				input: _image.output
-				command: name:    "python"
-				script: contents: #"print("Hello, world!")"#
+				// python:3.10-alpine has both python3 and python in the PATH
+				_image: docker.#Pull & {
+					source: "python:3.10-alpine"
+				}
+				run: python.#Run & {
+					input: _image.output
+					command: name:    "python3"
+					script: contents: #"print("Hello, world!")"#
```

Comment:
Nit: avoid raw strings: https://docs.dagger.io/1226/coding-style#avoid-raw-strings

Reply:
addressed in latest push